.taplo.toml

```diff
@@ -0,0 +1,34 @@
+include = ["**/Cargo.toml"]
+
+[formatting]
+align_entries = false
+align_comments = false
+array_trailing_comma = true
+array_auto_expand = false
+array_auto_collapse = false
+compact_arrays = true
+compact_inline_tables = false
+compact_entries = false
+column_width = 80
+indent_tables = false
+indent_entries = false
+indent_string = "    "
```

Comment:
What are the standard settings?
I started with the default options in taplo: https://github.com/tamasfe/taplo/blob/5cc5670f51f8c9928ce4d9f526f3175f86450024/crates/taplo/src/formatter/mod.rs#L140
I updated them to follows our conventions and not to change our files too much. Ex: indent_string= 4 whitespaces.

Reply:
A good, I think the default options are fine, but then I think it would be good to have the `taplo.toml` almost empty; no need to add these options explicitly. Then it is obvious to the reader that we are simply using the default. :)

However, I think this topic is very opinionated, so I don't want to enforce this.